pkg/controllers/managementuser/rbac/prtb_handler.go

```diff
@@ -297,3 +306,63 @@ func buildRule(resource string, verbs map[string]bool) rbacv1.PolicyRule {
 		APIGroups: []string{"*"},
 	}
 }
+
+func (p *prtbLifecycle) reconcilePRTBUserClusterLabels(binding *v3.ProjectRoleTemplateBinding) error {
+	/* Prior to 2.5, for every PRTB, following CRBs are created in the user clusters
+		1. PRTB.UID is the label key for a CRB, PRTB.UID=owner-user
+		2. PRTB.UID is the label value for RBs with authz.cluster.cattle.io/rtb-owner: PRTB.UID
+	Using this labels, list the CRBs and update them to add a label with ns+name of CRTB
+	*/
+
+	set := labels.Set(map[string]string{string(binding.UID): owner})
+	userCRBs, err := p.m.clusterRoleBindings.List(metav1.ListOptions{LabelSelector: set.String()})
+	if err != nil {
+		return err
+	}
+
+	for _, crb := range userCRBs.Items {
+		if crb.Labels[rtbLabelUpgraded] == "true" {
+			continue
+		}
+
+		retryErr := retry.RetryOnConflict(retry.DefaultRetry, func() error {
+			crbToUpdate, updateErr := p.m.clusterRoleBindings.Get(crb.Name, metav1.GetOptions{})
+			if updateErr != nil {
+				return updateErr
+			}
+			crbToUpdate.Labels[binding.Namespace+"_"+binding.Name] = owner
```

Comment:
Create a helper function for the `binding.Namespace+"_"+binding.Name` part. If it ever needs to change that will be a bummer to find all the places it's used.